CHANGELOG.md

```diff
@@ -5,7 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Gaprindashvili-2
+## Gaprindashvili-3
+
+### Added
```

Comment:
Pretty sure this should be "Removed" right?

Reply:
Thanks,...I'll fix. You get a little punchy doing these things. Good catch.